addons/web/static/src/views/fields/contact_statistics/contact_statistics.xml

```diff
@@ -2,10 +2,9 @@
 <templates xml:space="preserve">
 
     <t t-name="web.ContactStatisticsField">
-        <div class="d-flex flex-wrap gap-1">
-            <span t-foreach="list" t-as="data" t-key="data_index"
-                t-attf-class="badge bg-secondary rounded-pill smaller #{data.tagClass ? 'o_tag ' + data.tagClass : ''}" t-att-aria-label="data.label" t-att-title="data.label">
-                <i t-attf-class="fa me-1 {{data.iconClass}}" role="img"/>
+        <div class="d-inline-flex flex-wrap gap-3">
+            <span t-foreach="list" t-as="data" t-key="data_index" t-att-aria-label="data.label" t-att-title="data.label" t-att-data-tooltip="data.label">
+                <i t-attf-class="fa fa-fw me-1 pe-none {{data.iconClass}}" role="img"/>
```

Comment:
data.tagClass is no longer used. Could you clean up the definitions? See `_compute_application_statistics_hook`